_drafts/2017-05-16-lingtypology.Rmd

```diff
@@ -0,0 +1,199 @@
+---
+name: lingtypology
+layout: post_discourse
+title: Easy linguistic mapping with lingtypology
```

Comment:
Hello George @agricolamz,
I'm rOpenSci's Community Manager. I typically read over posts before they go up. Hope you don't mind me giving some feedback. After you respond to my comments, @sckott can give the final go-ahead and post.

First, thank you for writing this post. I especially like the intro where you define goals of linguistic mapping, and the narrative of the post makes very clear the cool features that `lingtypology` provides e.g. "...you can do incredible things and easily create maps that haven’t been created before...".